test/3x/torch/test_config.py

```diff
@@ -57,7 +57,7 @@ def test_quantize_rtn_from_dict_beginner(self):
         from neural_compressor.torch import quantize
 
         quant_config = {
-            "rtn_weight_only_quant": {
+            "RTN": {
```

Comment:
```suggestion
            "rtn": {
```
Should align with the `RTN` defined in common/utility.py.